client/src/modules/Mentor/pages/Interviews/components/RegistrationNotice.tsx

```diff
@@ -0,0 +1,48 @@
+import { Alert, Typography } from 'antd';
+import { InfoCircleTwoTone } from '@ant-design/icons';
+import moment from 'moment';
+import React from 'react';
+import { isInterviewRegistrationInProgess } from 'domain/interview';
+
+export function RegistrationNotice(props: {
+  name: string;
+  startDate: string;
+  showMentorOptions: (e: React.MouseEvent) => void;
```

Comment:
Please consider create separate type definitions for props and desctructure props in place `RegistrationNotice({ startDate, showMentorOptions, name }: Props) {...}`